.eslintrc.json

```diff
@@ -24,9 +24,7 @@
         "import/no-unresolved": "off",
```

Comment:
It actually work well but please let work without disabling our linters for good code practices